### DISCUSSION TITLE
Not on top of active window

### DISCUSSION THREAD

**Discussion by Senor-Charrleh** [SauceLLC/sauce4zwift#52] [General]
I can't get sauce to alway be on top unless zwift is in windowed mode... is this by designb? or should i be able to have it on top in full screen.

**Comment by suskozaver** [comment]
by design. 
Run Zwift in Windowed mode and use Borderless Gaming app to make it full screen.

**Comment by mayfield** [comment]
I would love it if I could do overlays in fullscreen mode, it's just not possible to do consistently on all platforms and there are bad side effects in the limited cases where I can support it (on macos).
